llama-index-integrations/readers/llama-index-readers-file/llama_index/readers/file/docs/base.py

```diff
@@ -28,6 +33,10 @@ def __init__(self, return_full_document: Optional[bool] = False) -> None:
         """
         self.return_full_document = return_full_document
 
+    @retry(
+        wait=wait_random_exponential(min=WAIT_MIN_SECONDS, max=WAIT_MAX_SECONDS),
+        stop=stop_after_attempt(RETRY_TIMES),
+    )
```

Comment:
Do you know if there's a specific error we should retry on? Right now this would retry on every type of error

I also wonder if the wait is even needed

Reply:
yup, lmc that

Reply:
@logan-markewich removed the wait, also we use lazy import so not sure if we would catch the exception if we import the package